src/logworker.cpp

```diff
@@ -120,6 +122,26 @@ namespace g3 {
       return std::unique_ptr<LogWorker>(new LogWorker);
    }
 
+   void LogWorker::removeWrappedSink(std::shared_ptr<g3::internal::SinkWrapper> sink) {
+      auto bg_removesink_call = [this, sink] {
+         _impl._sinks.erase(
+            std::remove(
```

Comment:
formatting nitpicking: can you put the `std::remove` args on the same line as the `std::remove`?